bytes-extended/src/main.sw

```diff
@@ -89,13 +115,16 @@ impl Bytes {
 
     /// Copies `byte_count` bytes from `bytes_ptr` into self at the specified offset.
     /// Reverts if the bounds of self are violated.
-    pub fn write_packed_bytes(ref mut self, offset: u64, bytes_ptr: raw_ptr, byte_count: u64) {
+    /// Returns the new length of self.
```

Comment:
Bc it's not changing the `len` of self, maybe `"Returns the byte offset immediately following what was written"` or something of that flavor?